vars/default_vars.yml

```diff
@@ -275,6 +275,10 @@ elgg_mysql_password: elgg4kids
 ejabberd_install: False
 ejabberd_enabled: False
 
+# Gitea
```

Comment:
Plz either:
1) remove this gratuitous line entirely, or...
2) Inject an inviting description, explaining why Gitea is attractive in ~10 words or less, similar to the Lokole description just below.